index.js

```diff
@@ -1,84 +1,170 @@
+require('dotenv').config();
```

Comment:
please remove this, if that's to be used, one must `-r dotenv/config` when calling node